argus/client/sct/client.py

```diff
@@ -148,7 +148,7 @@ def submit_performance_results(self, performance_results: PerformanceResultsRequ
         self.check_response(response)
 
     def create_resource(self, name: str, resource_type: str, public_ip: str, private_ip: str,
-                        region: str, provider: str, shards_amount: int, state=ResourceState.RUNNING) -> None:
+                        region: str, provider: str, dc_name: str, shards_amount: int, state=ResourceState.RUNNING) -> None:
```

Comment:
if we have region, that's in most cases 1to1 with dc_name, az would also be useful

Reply:
in most cases it's the same:
region <-> datacenter
availability zone <-> rack

but then we have fake racks, and soon also fake regions
so datacenter and rack, and we need precise about where we get the information from on SCT end.

Reply:
Done